doc/src/build/wallet.md

```diff
@@ -120,24 +144,48 @@ implement more secure key management and support hardware signing in a future re
 :warning: **Do not use in production**: Keys are stored in file!
 
 ## Starting the network
-Run the following command to start the local Sui network:
+Run the following command to start the local Sui network, assuming you
+accepted the default location for configuration:
 
 ```shell
-sui start
+$ sui start
 ```
 
-This command will by default look for Sui network coinfiguration file
-(`network.conf`) in the `~/.sui/sui_config` directory, but you can
-override this setting by providing a path to the directory where this
-file are stored:
+This command will look for the Sui network configuration file
+`network.conf` in the `~/.sui/sui_config` directory. But you can
+override this setting by providing a path to the directory where
+this file is stored:
+
+```shell
+$ sui start --config /path/to/sui/network/config/file
+```
+
+For example:
 
 ```shell
-sui start --config /path/to/sui/network/config/file
+$ sui start --config /Users/name/tmp/network.conf
+```
+
+TODO: Find out why I get this error:
+
+```
+claymurphy@Clays-MacBook-Pro tmp % sui start --config /Users/claymurphy/tmp/network.conf
+Error: Address already in use (os error 48)
+```
+
+I see no Sui-specific processes running:
+
+```
+claymurphy@Clays-MacBook-Pro tmp % ps -ef | grep -i sui
+  501   442     1   0 12:02PM ??         0:00.22 /System/Library/CoreServices/CoreServicesUIAgent.app/Contents/MacOS/CoreServicesUIAgent
+  501   623     1   0 12:03PM ??         0:01.10 /System/Library/PrivateFrameworks/AppleMediaServicesUI.framework/amsengagementd
+  501  1433     1   0  1:01PM ??         0:00.21 /System/Library/PrivateFrameworks/PaperKit.framework/Contents/LinkedNotesUIService.app/Contents/MacOS/LinkedNotesUIService
```

Comment:
I am fishing here - REST API server running locally?

Reply:
you can check which command is using the port using `lsof -i tcp:10000`